src/constants/indicators.js

```diff
@@ -179,59 +179,89 @@ export default {
     contextual: false,
     source: 'Education attainment - Completed Year 12, % of population',
   },
+  educationalAttainmentCertificate: {
+    name: 'Educational attainment - Certificate',
+    shortDescription: 'TODO',
+    longDescription: 'TODO',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0%',
+    contextual: false,
+    source: 'Education attainment - Completed Certificate III or IV, % of population',
+  },
+  educationalAttainmentTertiary: {
+    name: 'Educational attainment - Tertiary',
+    shortDescription: 'TODO',
+    longDescription: 'TODO',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0%',
+    contextual: false,
+    source: 'Education attainment - Completed Bachelor Degree or Higher, % of population',
+  },
   dispersion: {
     name: 'Local government dispersion',
-    shortDescription: 'Local government dispersion',
-    longDescription: 'Local government dispersion',
+    shortDescription: 'LGAs per 100,000 persons',
+    longDescription: 'Number of Local Government Areas per 100 000 persons',
     dataType: DATA_TYPES.NUMBER,
     aggregationMethod: AGGREGATION_METHODS.MEDIAN,
     format: '0[.]0%',
     contextual: false,
     source: 'Government Dispersion, Number of Local Government Areas per 100 000 persons',
   },
   dwellingHouse: {
-    name: 'House',
-    shortDescription: 'TODO',
-    longDescription: 'TODO',
+    name: 'Dwelling type ratio: house',
```

Comment:
There's colons here, but hyphens doing the same job up in "Educational attainment - Tertiary". Should they be the same?

Reply:
Colons are better, I'll standardise on that.